UD-Viz-Core/examples/data/ConfigData/DemoConfigData.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
Have a look at this [issue](https://github.com/VCityTeam/UD-Viz/issues/140) saying that we should change the general config file to a specific one for each demo. It would be nice to resolve it with your PR, since you seem to have an autonomous config file. Else, you should find a way to put your parameters in the global one, in order to stay consistent with other demos

Reply:
Files were changed to comply with this issue. The PR now partially solves issue 140, and enables us to have independent config files for each demo instead of having one general config file.